scripts/scraper/venv/lib/python3.12/site-packages/_pytest/_version.py

```
@@ -0,0 +1,34 @@
+# file generated by setuptools-scm
+# don't change, don't track in version control
+
+__all__ = [
+    "__version__",
+    "__version_tuple__",
+    "version",
+    "version_tuple",
+    "__commit_id__",
+    "commit_id",
+]
```

Ruff (0.14.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)